lib/constants.py

```diff
@@ -0,0 +1,3 @@
+LIGHT_SPEED = 299792458 #m/s
```

Comment:
please add a `.0`, otherwise python will use the number as integer - we know that the speed of light is an integer, but I fear that it might be used in an operation leading to a truncated integer result